checks/check-app-health.go

```diff
@@ -0,0 +1,59 @@
+package checks
+
+import (
+	"fmt"
+	"strings"
+	"time"
+	"net/http"
+	"io/ioutil"
+    "strconv"
+    "log"
+
+	maps "github.com/ashwanthkumar/golang-utils/maps"
+	"github.com/gambol99/go-marathon"
+)
+
+type AppHealth struct{}
+
+func (h *AppHealth) Name() string {
+	return "apphealth"
+}
+
+// Checks App health status of App Endpoint by reading router.hosts 
+//  and Check for 200 status code
+func (h *AppHealth) Check(app marathon.Application) AppCheck {
+	host := maps.GetString(app.Labels, "router.hosts", "notDefined")
```

Comment:
Should we skip this change if `router.hosts` is not defined? Also that parameter name is also not configurable. Can we please make it so? Given this is an open source versions, others might / might not want to enable this check. So we also need an ability to disable the check (via another label? -- which is disabled by default?) What are your thoughts on it?

Reply:
I think having another label to disable this check would be too verbose,  would it be possible to do this while invoking the app at the beginning?